### ISSUE TITLE
Missing set credentials on fs creation triggered by validator

### ISSUE THREAD

**Issue by thekoma** [GoogleCloudPlatform/hpc-toolkit#2684]
### Describe the bug

When creating filestore via OFE it fails silently because of ghpc validator:

This is triggered in at line https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/627b43aa3f238242b6ca98a5575541009ac411df/community/front-end/ofe/website/ghpcfe/cluster_manager/filesystem.py#L102 because is missing the correct env setup like this one https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/627b43aa3f238242b6ca98a5575541009ac411df/community/front-end/ofe/website/ghpcfe/cluster_manager/clusterinfo.py#L348

### Steps to reproduce

1. Create a filestore
2. it doesn't work

### Expected behavior

1. It creates the filestore
### Actual behavior
Fails silently and logs the error in `/opt/gcluster/hpc-toolkit/community/front-end/ofe/fs/fs_<id>/ghpc_create_log.stderr`.

### Version (`ghpc --version`)
main branch of this repository as today
### Blueprint

```yaml
blueprint_name: stg03

vars:
  project_id: myproject1298371289037
  deployment_name: stg03
  region: us-east1
  zone: us-east1-b
  labels:
    created_by: myproject1298371289037

deployment_groups:
- group: primary
  modules:
  - source: modules/file-system/filestore
    kind: terraform
    id: stg03
    settings:
      filestore_share_name: test
      network_id: projects/myproject1298371289037/global/networks/default
      zone: us-east1-b
      size_gb: 1024
      filestore_tier: BASIC_HDD
    outputs:
    - network_storage
```

### Output and logs

```log
Error: project "myproject1298371289037" does not exist or your credentials do not have permission to access it
Hint: It is possible the machine you are working on has not been authenticated.
Try to run `gcloud auth application-default login`

One or more blueprint validators has failed. See messages above for suggested
actions. General troubleshooting guidance and instructions for configuring
validators are shown below.

- https://goo.gle/hpc-toolkit-troubleshooting
- https://goo.gle/hpc-toolkit-validation

Validators can be silenced or treated as warnings or errors:

- https://goo.gle/hpc-toolkit-validation-levels
```

**Comment by cdunbar13** [comment]
Thank you raising this.  I will look into it and reach out if the team has any questions.

**Comment by cdunbar13** [comment]
One quick question, did you run `gcloud auth application-default login` with a user that has access to the project this is being deployed to?

**Comment by thekoma** [comment]
Hi @cdunbar13, thank you for your time.
No I didn't run `gcloud auth application-default login`.
I've used the ofe deploy script to create a VM in GCP with the webserver and everything needed via terraform.
Then I've create a SA and (as I'm in a demo env.) set it as owner of the project.
Then I've exported the SA key and imported it into the interface.

From what I see from documentation I should not interfere with the web application on the server side, everything should work via webui.
